test/e2e/build/_fixtures/simple/state1/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ubuntu:22.04
+
+RUN --mount=type=secret,id=ENV_SECRET cat /run/secrets/ENV_SECRET
```

Comment:
Let's also check the secret value. Something like `[ "$(cat /run/secrets/ENV_SECRET)" = "value" ]`.

Reply:
Let's hardcode it.

Reply:
done